source/chapters/appendix/settings_directory.rst

```diff
@@ -8,28 +8,26 @@ The Mixxx Settings Directory
 .. sectionauthor::
    ronso0 <ronso0@mixxx.org>
 
-The Mixxx settings directory contains all user data and settings of your Mixxx installation.
+The Mixxx settings directory contains all user data and settings of your Mixxx installation. If you want to make a backup of your library and all preferences copy the entire directory to a safe place.
 
 Location
 --------
 
-**Linux**
-
-    ``~/.mixxx/``
-
-    **Note:** make sure you have "Show hidden files" enabled in your file manager
-    in order to show all dot-files and directories.
+.. hint:: On Windows, the :file:`AppData` folder is hidden, as well as all files and
+          directories beginning with a dot '.' on Linux. So if you have not already,
+          you will need to set your file manager to show hidden files and folders.
 
 **Windows**
-
-    | Vista and up: :file:`%LOCALAPPDATA%\\Mixxx`
-    | XP and below: :file:`%USERPROFILE%\\Local Settings\\Application Data\\Mixxx\\`
-    | Type either of those into the location bar of a Computer or Folder window.
+ | Vista and up: ``%LOCALAPPDATA%\Mixxx``
```

Comment:
@JoergAtGithub Must admit I lost track of which paths works and which don't.
Could you please verify this works?
(related https://github.com/mixxxdj/manual/pull/439/files)

Reply:
The % is scary for some users. Windows users are used to paths starting with a drive letter. So we may add a bit of context here. e.g:

When pasting this path to the windows explorer, the %LOCALAPPDATA% part automatically replaced with your personal application data folder e.g.
C:\Users\username\AppData\Local.

Reply:
> @JoergAtGithub Must admit I lost track of which paths works and which don't. Could you please verify this works? (related https://github.com/mixxxdj/manual/pull/439/files)

Yes, this is described correct and works for me on Win7!